features/facebookImport/src/facebookImporter.jsx

```diff
@@ -90,32 +83,15 @@ const FacebookImporter = () => {
                     <Route exact path="/report">
                         <ReportView />
                     </Route>
-                    <Route exact path="/report/data-structure-info">
-                        <DataStructureInfoScreen />
-                    </Route>
-                    <Route exact path="/report/pictures-info">
-                        <PicturesInfoScreen />
-                    </Route>
-                    <Route exact path="/report/reaction-types-info">
-                        <PostReactionInfoScreen />
-                    </Route>
                     <Route exact path="/report/details">
```

Comment:
All this code saving is great, but I'm not entirely sure how this is being substituted by the popUps.